.appends/.github/labels.yml

```diff
@@ -0,0 +1,336 @@
+# ----------------------------------------------------------------------------------------- #
+# These are the repository-specific labels that augment the Exercise-wide labels defined in #
+# https://github.com/exercism/org-wide-files/blob/main/global-files/.github/labels.yml.     #
+# ----------------------------------------------------------------------------------------- #
+
+- name: "track/05ab1e"
+  description: "05AB1E track"
+  color: "cccccc"
```

Comment:
Such sad labels 😬 I don't know exactly which tool chose the color (either GH or the Labeler, when the Labeler tried to add labels to PRs that didn't exist). Can we bring back the previous color? Before they were accidentally deleted they were `#c8d7ff`. I know this is super crucial for this PR 😇 #sarcasm

Reply:
@angelikatyborska @SleeplessByte and I couldn't remember the color. I've updated!

I also add a missing `type/copy` label.